flytekit/core/type_engine.py

```diff
@@ -2256,6 +2301,17 @@ def _check_and_covert_float(lv: Literal) -> float:
     raise TypeTransformerFailedError(f"Cannot convert literal {lv} to float")
 
 
+def _check_and_covert_int(lv: Literal) -> int:
```

Comment:
1. yes I'm going to provide a more meaningful name
2. If we use protobuf struct as input from the console, this can't be deleted.
We have to provide an identifier to the console, then the console knows it can generate msgpack idl.